index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TechSplendor</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="stylesheet" href="./css/desktop.css">
+
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+  <script defer src="./js/script.js"></script>
+</head>
+
+<body>
+  <header>
+    <nav class="mobile-nav">
+      <img src="./images/mobile-menu-icon.png" alt="Mobile menu icon" class="mobile-menu-icon">
+      <a href="./index.html"><img src="./images/tech-splendor.png" alt="Project logo" class="project-logo"></a>
+
+      <div class="mobile-menu">
+        <img src="./images/close-menu.png" alt="Close mobile menu icon" class="close-menu-icon">
+        <ul class="menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#">TS Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <nav class="desktop-nav">
+      <ul class="socials">
+        <li><a href=""><img src="./images/socials/facebook.svg" alt="Facebook icon"></a></li>
+        <li><a href=""><img src="./images/socials/twitter.svg" alt="Twitter icon"></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+
+      <div>
+        <a href="./index.html">
+          <img src="./images/tech-splendor.png" alt="Project Logo" class="desktop-project-logo">
+        </a>
+        <ul class="menu desktop-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#">TS Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section class="hero">
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly pay attention style of `right border` to `nav` menues and give background picture to `Home` page like a [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

### Desktop
![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/63915024/d9576265-9864-4267-b95b-2e1a25a02f14)

### Mobile 
![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/63915024/15758009-a9b6-4cad-aab1-2c319e9395e4)